products/revenue_analytics/backend/hogql_queries/revenue_analytics_insights_query_runner.py

```diff
@@ -178,6 +180,92 @@ def _get_subquery_by_product(self) -> ast.SelectQuery | None:
 
         return query
 
+    def _get_subquery_by_country(self) -> ast.SelectQuery | None:
```

Comment:
Yes, this is a bunch of copy-paste, https://github.com/PostHog/posthog/issues/33502 is supposed to fix this with a much more generic interface